kanidmd/src/lib/idm/server.rs

```diff
@@ -149,6 +150,7 @@ impl IdmServer {
         origin: String,
         // ct: Duration,
     ) -> Result<(IdmServer, IdmServerDelayed), OperationError> {
+        let _entered = trace_span!("IdmServer::new").entered();
```

Comment:
`trace_span!()` doesn't require specifying the target, it only requires a name for the span,. I just happened to arbitrarily chose to be the the name of the function because I thought it was the most descriptive.

Reply:
So, what's the functional difference between `trace_span` and `span`, when we're using it to trace the function the same way span does? 😄

Reply:
With `span`, you have the specify the level, and `trace_span` is just shorthand for specifying `span` at the trace level.

https://docs.rs/tracing/0.1.26/tracing/index.html#shorthand-macros

Reply:
Thanks 😄